meerk40t/core/elements.py

```diff
@@ -5052,13 +5065,16 @@ def load(self, pathname, **kwargs):
                 if str(pathname).lower().endswith(extensions):
                     try:
                         results = loader.load(self.context, self, pathname, **kwargs)
+                        if results and isinstance(results, Node):
+                            results.focus()
+                            self.classify([elem.object for elem in results.flat(types=("elem"))])
                     except FileNotFoundError:
```

Comment:
Results is going to be pretty much always True here, since the loading function is returning that rather than the loaded data. I am not sure what this is otherwise supposed to do since results shouldn't be a node.

Reply:
I changed the loaders to return the `file node` if they create it and elements (svg, dxf, image) or `True` if they don't.

I could classify the elements in the loader and just return True like before, but at the time (late at night - judgement may have been poor) it seemed better than having the code in the loader itself. But on reflection I think it would be better to switch back and to simply find a better way to do classify in svg_io (which is not hard).

So I am not going to merge this - but will add more commits to redo it.